docs/UserGuide.md

```diff
@@ -92,6 +92,7 @@ Adds a new entry to their address book for financial advisors, including persona
 * Email address: Valid string and email address format (johnd@example.com).
 * Occupation: Valid string occupation.
 * Tag: Valid string.
+* Appointment Date: Only accepted in yyyy-mm-dd HH:mm, mm/dd/yyyy HH:mm or dd-mm-yyyy HH:mm format
```

Comment:
Is there any particular reason for the "D" of Date is capitalized here? Since the rest like the "n" for Phone number and "a" for Email address are not capitalized.